include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -179,11 +179,12 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
   moveit::core::RobotModelConstPtr getRobotModel();
 
   /**
-   * \brief Call this once at begining to load the robot marker
-   * \param ee_jmg - the set of joints to use, e.g. the MoveIt! planning group, e.g. "left_arm"
+   * \brief Call this once at begining to load the end effector markers and then whenever a joint changes
+   * \param ee_jmg - the set of joints to use, e.g. the MoveIt! planning group, e.g. "left_gripper"
+   * \param ee_joint_pos - the values of all active joints in this planning group
    * \return true if it is successful
    */
-  bool loadEEMarker(const robot_model::JointModelGroup* ee_jmg);
+  bool loadEEMarker(const robot_model::JointModelGroup* ee_jmg, const std::vector<double>& ee_joint_pos);
```

Comment:
Would you like to have a default value here? just noticed that I changed a public api in a non backwards compatible manner here. Would you like a default value " const std::vector<double>& ee_joint_pos = {} " here?

Reply:
ping?

Reply:
yes please, or provide two versions of the function